Dockerfile

```diff
@@ -22,10 +13,18 @@ ENV VARNISH_VARNISHD_PARAMS ''
 ENV VARNISH_VARNISHNCSA_PARAMS ''
 ENV VARNISH_CACHE_TAGS_HEADER Cache-Tags
 
+COPY patches /tmp/patches/
+COPY config /tmp/config/
+COPY scripts /tmp/scripts/
```

Comment:
Use `/opt/scripts` instead

Reply:
all files placed in /tmp are only needed during the build. after the build finished, everything in the /tmp directory is deleted. I do not see any reason to put the files necessary only for build process in any place other than the tmp directory